test/integration/data.js

```diff
@@ -274,6 +274,35 @@ module.exports = {
       two: instance('binaryType', 'two', '<file2>here_is_file2.jpg</file2>'),
       both: instance('binaryType', 'both', '<file1>my_file1.mp4</file1><file2>here_is_file2.jpg</file2>')
     }
-  }
+  },
+  simpleWithLongFormId: `<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa">
```

Comment:
rather than create an entire new blob of XML for this case, i would prefer you just replace the string inline with the test. i do something similar here: https://github.com/opendatakit/central-backend/blob/issa/encryption/test/integration/api/submissions.js#L599

Reply:
I thought we will need new blob as it is a form and not a submission instance. In current test data, we don't have a function that can generate form with required `id`. Please let me know if I am missing anything.

Reply:
i broke the link, but as you can see here:
https://github.com/opendatakit/central-backend/blob/7aaab0f/test/integration/api/submissions.js#L685

works just fine with forms.